tests/components/tibber/test_services.py

```diff
@@ -119,15 +120,13 @@ def create_mock_hass():
     return mock_hass
 
 
-def remove_microseconds(dt):
-    """Remove microseconds from a datetime object."""
-    return dt.replace(microsecond=0)
-
-
-async def test_get_prices():
+async def test_get_prices(
+    freezer: FrozenDateTimeFactory,
+) -> None:
     """Test __get_prices with mock data."""
-    today = remove_microseconds(dt.datetime.now())
-    tomorrow = remove_microseconds(dt.datetime.now() + dt.timedelta(days=1))
```

Comment:
Did we remove microseconds because the service call takes in a date in isoformat?

Reply:
i guess it was to avoid differences in microseconds, because the processing time could not be assumed?